addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2090,7 +2090,7 @@ const userValueWidgetsRegistry = {
 const SnippetOptionWidget = Widget.extend({
     tagName: 'we-customizeblock-option',
     events: {
-        'click .o_we_collapse_toggler': '_onCollapseTogglerClick',
+        'click .o_we_collapse_toggler, .o_we_toggler': '_onCollapseTogglerClick',
```

Comment:
Why not using the o_we_collapse_toggler instead of creating a new one?

Reply:
if i use `.o_we_collapse_toggler` i have to replace all css that contain, so i decided to create a new class.

Reply:
Ok, then instead of this, just confine the scss to `we-toggler.o_we_collapse_toggler`

So, only one class: `.o_we_collapse_toggler` which has a JS handler attached to it
And the style you talked about is only applied to `we-toggler.o_we_collapse_toggler`